lib/fog/google/requests/storage_json/put_bucket.rb

```diff
@@ -0,0 +1,69 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # Create an Google Storage bucket
+        #
+        # ==== Parameters
+        # * bucket_name<~String> - name of bucket to create
+        # * options<~Hash> - config arguments for bucket.  Defaults to {}.
+        #   * 'LocationConstraint'<~Symbol> - sets the location for the bucket
+        #   * 'x-amz-acl'<~String> - Permissions, must be in ['private', 'public-read', 'public-read-write', 'authenticated-read']
+        #
+        # ==== Returns
+        # * response<~Excon::Response>:
+        #   * status<~Integer> - 200
+        def put_bucket(bucket_name, options = {})
+          acl = options["x-goog-acl"] || "private"
+          location = options["LocationConstraint"] || nil
+
+          api_method = @storage_json.buckets.insert
+          parameters = {
+            "project" => @project,
+            "predefinedAcl" => acl,
+            "predefinedObjectAcl" => acl,
+            "projection" => "full"
+          }
+          body_object = {
+            :name => bucket_name,
+            :location => location
+          }
+
+          request(api_method, parameters, body_object = body_object)
+        end
+      end
+
+      class Mock
+        def put_bucket(bucket_name, options = {})
+          acl = options["x-goog-acl"] || "private"
+          if !%w(private publicRead publicReadWrite authenticatedRead).include?(acl)
+            raise Excon::Errors::BadRequest.new("invalid x-goog-acl")
+          else
+            data[:acls][:bucket][bucket_name] = self.class.acls(options[acl])
+          end
+          response = Excon::Response.new
+          response.status = 200
+          bucket = {
+            :objects        => {},
+            "Name"          => bucket_name,
+            "CreationDate"  => Time.now,
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.